diagnostic_rt_qpcr/libraries/protocolsurvey/source.rb

```diff
@@ -0,0 +1,63 @@
+# frozen_string_literal: true
+
+# Includes methods for giving a survey to the Tech and get ting answers.
+
+module ProtocolSurvey
+
+  def protocol_survey(operations)
+    question1 = 'Are you aware of any mistakes or errors that may have occurred '\
+                'while executing this Job?  If so briefly explain.'
+    survey = show do
+      title 'Protocol Survey'
+      note 'Please answer the following question to the best of your ability'
+      separator
+      note question1
+      get('text',
+          var: question1,
+          label: "Put 'NA' if not applicable",
+          default: '')
+    end
+    answers = [{question: question1, response: survey.get_response(question1)}]
+    associate_answers(operations: operations, answers: answers)
+  end
+
+  def workflow_survey(operations)
+    question1 = 'Were all the instructions for this job clear? '\
```

Comment:
Be good to use unique keys with versioning, i.e., `question1_ver1` if the question wording is not captured. And don't use the same question number over, i.e., `question1` is always for versions of one question